src/connectedk8s/azext_connectedk8s/_utils.py

```diff
@@ -52,6 +62,7 @@ def get_chart_path(registry_path, kube_config, kube_context):
 
     # Exporting helm chart
     chart_export_path = os.path.join(os.path.expanduser('~'), '.azure', 'AzureArcCharts')
+    cleanup_helm_chart_directory(chart_export_path)
```

Comment:
>cleanup_helm_chart_directory [](start = 4, length = 28)

Why not add this within the export_helm_chart itself?

Reply:
done